Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst

```diff
@@ -171,16 +171,20 @@ The configuration looks like this:
       CategoryListing:
         type: Simple
         limitToPages: [13]
-        routePath: '/show-by-category/{category_id}/{tag}'
+        routePath: '/show-by-category/{category}/{tag}'
         defaults:
           tag: ''
         requirements:
-          category_id: '[0-9]{1,3}'
+          category: '[0-9]{1,3}'
           tag: '[a-zA-Z0-9]+'
         _arguments:
           category_id: 'category'
 
-:yaml:`routePath` defines the static keyword and the placeholders.
```

Comment:
":aspect:" is not a directive we use anywhere.

`category_id` makes it more clear that a ID is used, instead of a "category name". We should keep that.

Explaining "tag" and "category_id" here is good, but I would rather use the sub keys to reference it.

Reply:
Yes, `category_id` makes it clear, but it is **wrong**! This won't work.

Reply:
I have just committed 3 new versions which should solve the miss-understandings. So, IMO this is solved now.

Reply:
Sorry, may false. unresolved that issue. For me `:aspect:` ist just an optical "highlight" like described here: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Reference/Code/InlineCode.html#text-roles
So yes, I also would prefer `:yaml:` here.

Reply:
OK...got it, because one single word is not valid YAML we should use another directive to highlight such words. Further, syntax checks and parsing of yaml directives may fail (in future), if they contain invalid yaml.

Reply:
I don't really understand, could you explain this to me? `:yaml:` is used many times (154x) in this manual alone, even for single words, what's wrong about it?

Reply:
When you use the text role it doesnt have to be a complete valid yaml file. That is only true for code-blocks

Reply:
So the yaml text role is correct here